servicetalk-opentelemetry/src/test/resources/services/io.opentelemetry.context.ContextStorageProvider

```diff
@@ -0,0 +1 @@
+io.servicetalk.opentelemetry.asynccontext.ServiceTalkContextScopeManager
```

Comment:
This file likely won't be required for tests if you split modules and add `servicetalk-opentelemetry-asynccontext` as a dependency.

Reply:
true, deleted that.